css/main.css

```diff
@@ -69,8 +74,37 @@ input {
     margin: 10px;
     border-radius: 11px;
     border: 2px solid white;
-    font: bold Arial;
+    font: bold Arial, Helvetica, sans-serif;
     font-size: 25px;
     text-align: center;
     color: white;
 }
+
+.digit-button:hover{
+    border: 2px solid orchid;
+} 
+ 
+.digit-button:active{
+    border: 2px solid orchid;
+    color: orchid;
+}   
+
+.math-operation-button:hover{
```

Comment:
move `.math-operation-button:hover` and `.math-operation-button:active` selectors near to `.math-operation-button` selector.